### DISCUSSION TITLE
Can Fooocus Do Image to Image?

### DISCUSSION THREAD

**Discussion by cerrec** [lllyasviel/Fooocus#187] [Q&A]
I would love to transform my own images into the styles that Fooocus provides. Is there a way to do this?

**Comment by y0himba** [comment]
From what I can see, it doesn't.  However I would love to second this feature.  I haven't been this excited for a SD interface in a while.  Fooocus is exactly what the AI ordered!

**Comment by barepixels** [comment]
Use this fork for img2img https://github.com/MoonRide303/Fooocus-MRE
